### PR TITLE
Update .gitignore && add dockerApps.evaluation-test to CI

### PR DESCRIPTION
**Pull request by zeme-wana**
Things that slipped through the previous PR

### Summary by CodeRabbit

**Refactor:**
- Updated the path for Docker app in `nix/ci.nix`.
- Modified the values of `genFlake` and `buildFlake` variables in `nix/docker-files/docker.nix`.
- Changed the `defaultPackage` attribute value in `nix/per-system-outputs.nix`.

**Chore:**
- Added `.pre-commit-config.yaml` to `.gitignore`.

> 🐇💻
> 
> "In the land of code, where logic intertwines,
> A rabbit hops along the lines.
> Paths are changed, variables renamed,
> Yet the essence of the code remained.
> With every commit, we refine,
> In this digital world, we shine! 🌟"

### CODE REVIEW

File: nix/docker-files/docker.nix [hunk 1]
```diff
@@ -64,8 +64,8 @@
       finalImageName = "nixos/nix";
       finalImageTag = "2.15.0";
     };
-    genFlake = inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927;
```
Comment (zeme-wana) [inline nix/docker-files/docker.nix:67]: Aliases for your cabal *executables* only (this excludes benchmarks and test suites) are added to your `packages` and `apps` flake outputs.
Alias means just the executable name as found in your cabal file, without the `$cabal_package_name-exe-` prefix.

For projects supporting a single compiler (like in this repo, `ghc927`), the compiler name is no longer appended to the flake outputs names.

```
inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927
->
inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927

inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927
->
inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927

inputs'.self.apps.dapps-certification-exe-plutus-certificatio-ghc927
->
inputs'.self.apps.plutus-certification
```

If you cabal project has multiple executables with the same name across your cabal files, a warning will be generated when running `list-flake-outputs` and no aliases will be created.

**Review by bogdan-manole — APPROVED** [review]
lgtm

**Comment by zeme-wana** [comment]
@bogdan-manole are we ok with the `CI Tests / tests` failing? I see that they are red on the `main` branch too.

**Comment by bogdan-manole** [comment]
> @bogdan-manole are we ok with the `CI Tests / tests` failing? I see that they are red on the `main` branch too.

yes, that's ok, anyway `react` part will be removed from this repo

**Pull request merged**